### DISCUSSION TITLE
metasploit-latest-windows-x64-installer ,这个安装后，不就是启动不了，界面。不知道怎么弄，

### DISCUSSION THREAD

**Discussion by 2773308162** [rapid7/metasploit-framework#19207] [General]
metasploit-latest-windows-x64-installer ,这个安装后，不就是启动不了，界面。不知道怎么弄，

**Comment by adfoster-r7** [comment]
metasploit-framework is free and open source, there are multiple tutorials available online showing how to use this tool 👍
